src/phoenix/trace/semantic_conventions.py

```diff
@@ -38,6 +38,8 @@ class DeploymentAttributes(AttributeGroup):
     }
 
 
+EXCEPTION_MESSAGE = "exception.message"
```

Comment:
If we are following OTEL, this should be added to the events as an exception event: https://github.com/Arize-ai/phoenix/blob/e5f716e1188b5289cc28d2c462952b8a4dc780e1/src/phoenix/trace/schemas.py#L65